spec/features/user_creates_orders_spec.rb

```diff
@@ -17,12 +17,12 @@
 
         user = User.create(username: "adam", password: "adam")
         d = Destination.create(continent: "North America", description: "North American continent description.",
-                            image_url: "south-america.jpg" )
+        image_url: "south-america.jpg" )
         a = Activity.create(name: "Hiking", description: "We loven to do ze hikes.", image_url: "hiker.jpg")
         Trip.create(destination_id: d.id, activity_id: a.id, trip_cost: 10000.00, name: "Canadian Rockies")
         dd = Destination.create(continent: "South America", description: "South American continent description.",
-                            image_url: "south-america.jpg" )
-        Trip.create(destination_id: dd.id, activity_id: a.id, total_cost: 400000.00, trip_cost: 15000.00, name: "Patagonia" )
+        image_url: "south-america.jpg" )
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside parentheses detected.